ga4gh/converters.py

```diff
@@ -0,0 +1,43 @@
+"""
+Provides classes that take protocol requests, send that request to
+the server, and write a particular genomics file type with the results.
+"""
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+
+class AbstractConverter(object):
+    """
+    Abstract base class for converter classes
+    """
+    def __init__(self, request, outputStream):
+        self.request = request
+        self.outputStream = outputStream
+
+    def close(self):
```

Comment:
I don't think we're ever going to close a stream, since we've passed it in open. The canonical usage would be
```python
with open(fileName, "w") as outputFile:
    converter = VcfConverter(request, outputFile)
    converter.convert()
```